structuredefinitions/UKCore-Specimen.xml

```diff
@@ -143,6 +143,16 @@
       </slicing>
       <min value="0" />
     </element>
+    <element id="Specimen.collection.extension:specimenCollectionCollectorR5">
+      <path value="Specimen.collection.extension" />
+      <sliceName value="specimenCollectionCollectorR5" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Specimen.collection.collector" />
+      </type>
+      <isModifier value="false" />
```

Comment:
As discussed, is missing its end of element tag

Reply:
And do you want to add a short description to the value[x] in the extension in the profile to force it to render?

Reply:
done